src/explorer/helpTree.ts

```diff
@@ -97,26 +97,21 @@ export default class HelpTree
 
       const feedback = new HelpLinkTreeItem(
         'Suggest a Feature',
-        'https://feedback.mongodb.com/forums/929236-mongodb-for-vs-code/',
+        LINKS.feedback,
         'feedback',
         'lightbulb'
       );
 
       const reportBug = new HelpLinkTreeItem(
         'Report a Bug',
-        'https://github.com/mongodb-js/vscode/issues',
+        LINKS.reportBug,
         'reportABug',
         'report'
       );
 
-      const telemetryUserIdentity =
-        this._telemetryService?.getTelemetryUserIdentity();
-      const ajsAid = telemetryUserIdentity
-        ? `&ajs_aid=${telemetryUserIdentity[0]}`
-        : '';
       const atlas = new HelpLinkTreeItem(
         'Create Free Atlas Cluster',
-        `https://mongodb.com/products/vs-code/vs-code-atlas-signup?utm_campaign=vs-code-extension&utm_source=visual-studio&utm_medium=product${ajsAid}`,
```

Comment:
With the change, we will be missing out on `utm_campaign` and `ajsAid` query params. Just confirming if this is intentional?

Reply:
Yes, in the ticket:

```
There are some links that already have UTMs, like the "Create Free Atlas Cluster" link below, they can all be standardized to be the same:

https://www.mongodb.com/products/vs-code/vs-code-atlas-signup?utm_campaign=vs-code-extension&utm_source=visual-studio&utm_medium=product&ajs_aid=undefined

can become:

https://www.mongodb.com/products/vs-code/vs-code-atlas-signup?utm_source=vscode&utm_medium=product
```